stdlib/numbers.pyi

```diff
@@ -102,6 +102,7 @@ class Complex(Number, _ComplexLike):
     def conjugate(self) -> _ComplexLike: ...
     @abstractmethod
     def __eq__(self, other: object) -> bool: ...
+    __hash__: ClassVar[None]  # type: ignore[assignment]
```

Comment:
The `Number` class states that ["Concrete numeric types must provide their own hash implementation"](https://github.com/python/cpython/blob/a25042e6d2b71651e93ff90c361ad763c0368872/Lib/numbers.py#L45-L46)

Feel free to enlighten me if I've missed something, but doesn't the change here mean that you can't actually do that? So for example, doesn't this line mean that the following wouldn't work?
```python
from numbers import Complex

class Test(Complex):
	def __hash__(self) -> int:
		return 0
```